### DISCUSSION TITLE
It's getting there.

### DISCUSSION THREAD

**Discussion by ThatRetroGuy** [skjelten/emusc#68] [General]
Ive tried some midi file's on a program that would play fine on the nuked-sc,  tried playing a classic music piece and the ocarina for example on this tune is slightly off key, same with the church organ sound, i use midi player from falcosoft.
its like that one muscian who can play a tune right, but its a little off-key.

**Comment by skjelten** [comment]
I probably mentioned this in another post, but my guess is that the lack of TVF (filtering) is the key to why it feels a bit off. Next release will have the TVF in place (I hope), so then we will know.

**Reply by ThatRetroGuy** [reply]
i hope that fixes it, its gotten a long way since, and ive kind of hit a little road block, for some reason it error's out on wav, and refuses to start the emulator after.

**Reply by ThatRetroGuy** [reply]
Oh, by the way, do you have an email? wanna tell something, want to keep the situation low on github.

**Reply by skjelten** [reply]
Have you checked the troubleshooting guide at the wiki? See [here](https://github.com/skjelten/emusc/wiki/Troubleshooting-Guide). 

Feel free to contact me at my private email: github account name @pvv.org
